wdl-ast/src/lib.rs

```diff
@@ -55,11 +55,13 @@ pub use wdl_grammar::ToSpan;
 pub use wdl_grammar::WorkflowDescriptionLanguage;
 
 pub mod v1;
+pub mod version;
 
 mod validation;
 mod visitor;
 
 pub use validation::*;
+pub use version::SupportedVersion;
```

Comment:
This is where I pulled the `Version` out of validation and called it `SupportedVersion`.